src/components/Dialog/Dialog.st.css

```diff
@@ -0,0 +1,72 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+:import {
+    -st-from: "../IconButton/IconButton.st.css";
+    -st-default: TPAIconButton;
+}
+
+/*Defaults*/
+:vars {
+    DefaultCloseButtonColor: color-5;
+}
+
+/*Overrides*/
+:vars {
+    CloseButtonColor: --overridable;
+}
+
+/*Fallbacks*/
+:vars {
+    CurrentCloseButtonColor: color(fallback(value(CloseButtonColor), value(DefaultCloseButtonColor)));
+}
+
+.root {
+    -st-states: mobile;
+}
+
+.root:mobile {
+}
+
+.contentWrapper {
+    width: 100%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -30%);
+
+    background: #fff;
+}
+
+.root:mobile .contentWrapper {
+    height: 100%;
+    transform: translate(-50%, -50%);
+}
+
+.closeButtonWrapper {
+    position: absolute;
+    top: 20px;
+    right: 20px;
+}
+
+:global([dir="rtl"]) .closeButtonWrapper,
+:global(.rtl) .closeButtonWrapper {
```

Comment:
So actually for `rtl` cases, we use the `rtl` prop that's passed through the context.
Just like the `mobile` prop.
and then we can use it as a Stylable state.
You can see an example here: https://github.com/wix/wix-ui-tpa/blob/291f717d564cffd107f9d194ba9feca3f6d35c9b/src/components/Pagination/Pagination.tsx#L26